src/main/java/io/stargate/sgv2/jsonapi/api/model/command/Command.java

```diff
@@ -52,4 +56,13 @@
   @JsonSubTypes.Type(value = UpdateManyCommand.class),
   @JsonSubTypes.Type(value = UpdateOneCommand.class),
 })
-public interface Command {}
+public interface Command {
+  /**
+   * Default method if (and only if) there is no "options" property. Used for allowing all Commands
+   * to take in empty Options without having to add Record field if none accepted.
+   */
+  @JsonProperty("options")
+  default void options(Map<String, Object> optionsIgnored) {
+    // Should we fail for non-empty options?
```

Comment:
I think individual commands should decide which options are valid or not. It's just empty options should be fine for all commands.

Reply:
@vkarpov15 Agreed. This only applies to `Command`s that do not declare that they take `options` parameter -- which is the default "only empty accepted" case. Ones that do take it would not call this method.

So the question is that if `Command` is not marked/defined to take Options, then should we verify that only missing/empty JSON Object options are allowed and give error for others.
I'm fine with no error, it's a philosophical choice of leniency of arguments.